common/static/coffee/src/discussion/utils.coffee

```diff
@@ -113,7 +113,7 @@ class @DiscussionUtil
       alertDiv = $("<div class='modal' role='alertdialog' id='discussion-alert' aria-describedby='discussion-alert-message'/>").css("display", "none")
       alertDiv.html(
         "<div class='inner-wrapper discussion-alert-wrapper'>" +
-        "  <button class='close-modal dismiss' aria-hidden='true'><i class='icon fa fa-times'></i></button>" +
+        "  <button class='close-modal dismiss' aria-hidden='true'><span class='icon fa fa-times' aria-hidden='true'></span></button>" +
```

Comment:
@clrux Shouldn't this have sr text for the close icon? This is the dialog in action:

![image](https://cloud.githubusercontent.com/assets/484484/15624112/582896e4-244d-11e6-9ebf-03678be51fcf.png)

And why is the button aria-hidden?